solutions/cses-1662.mdx

```diff
@@ -144,7 +144,7 @@ for a in Z:
 ans = 0
 for x in M:
 	# Calculate the # of pairs w/o duplicates & reverse groups.
-	ans+= x*(x-1) // 2
+	ans+= x*(x-1)//2
```

Comment:
wha, who told you to do this

Reply:
ok I will added them again 
please do merge